src/tools/clippy/tests/ui/map_flatten_fixable.fixed

```diff
@@ -59,8 +59,6 @@ fn issue8878() {
         .and_then(|_| {
 // we need some newlines
 // so that the span is big enough
-// we need some newlines
-// so that the span is big enough
```

Comment:
Those lines are probably there in order to test that also with more than 6 lines the message is displayed correctly. Can you add those back an re-bless the tests?

Reply:
That's weird, because I didn't actually change anything here, the original file doesn't have these too: https://github.com/rust-lang/rust/blob/10f4ce324baf7cfb7ce2b2096662b82b79204944/src/tools/clippy/tests/ui/map_flatten_fixable.rs#L56-L67

These were removed by `x test clippy --bless`

Reply:
The commit that added this test, https://github.com/rust-lang/rust-clippy/commit/22673bce4cd8bad876d047172680a9ced02ba4c1 for some reason have more comments in the [`.fixed`](https://github.com/rust-lang/rust-clippy/commit/22673bce4cd8bad876d047172680a9ced02ba4c1#diff-ca7b9a9741c1ed12c2fd24100b1c5a7e019391351f0514d89d285e09c76e91c9R41-R45) file, than in the [`.rs`](https://github.com/rust-lang/rust-clippy/commit/22673bce4cd8bad876d047172680a9ced02ba4c1#diff-e3be7aabb3c932cd29ab37731eb22808525308ab0a3e4f79aabc92633df3eb14R41-R43) file.

Reply:
Oh so this was a bug before, but this PR now fixed that bug. Awesome! Thanks for pointing this out.